app/components/work_packages/activities_tab/journals/item_component.rb

```diff
@@ -0,0 +1,153 @@
+            "application-target": "dynamic",
+            "work-packages--activities-tab--item-activity-url-value": activity_url
+          }
+        end
+
+        def show_comment_container?
+          journal.notes.present? && filter != :only_changes
+        end
+
+        def activity_url
+          "#{project_work_package_url(journal.journable.project, journal.journable)}/activity#{activity_anchor}"
+        end
+
+        def activity_anchor
+          "#activity-#{journal.version}"
+        end
+
+        def editable?
+          journal.user == User.current
+        end
+
+        def updated?
+          return false if journal.initial?
+
+          journal.updated_at - journal.created_at > 5.seconds
+        end
+
+        def has_unread_notifications?
+          journal.notifications.where(read_ian: false, recipient_id: User.current.id).any?
+        end
+
+        def notification_on_details?
+          has_unread_notifications? && journal.notes.blank?
+        end
+
+        def copy_url_action_item(menu)
+          menu.with_item(label: t("button_copy_link_to_clipboard"),
+                         tag: :button,
+                         content_arguments: {
+                           data: {
+                             action: "click->work-packages--activities-tab--item#copyActivityUrlToClipboard"
+                           }
+                         }) do |item|
+            item.with_leading_visual_icon(icon: :copy)
+          end
+        end
+
+        def edit_action_item(menu)
+          menu.with_item(label: t("js.label_edit_comment"),
+                         href: edit_work_package_activity_path(journal.journable, journal, filter:),
+                         content_arguments: {
+                           data: { "turbo-stream": true, test_selector: "op-wp-journal-#{journal.id}-edit" }
+                         }) do |item|
+            item.with_leading_visual_icon(icon: :pencil)
+          end
+        end
+
+        def quote_action_item(menu)
+          menu.with_item(label: t("js.label_quote_comment"),
+                         tag: :button,
+                         content_arguments: {
+                           data: {
+                             action: "click->work-packages--activities-tab--index#quote",
+                             "content-param": journal.notes,
+                             "user-name-param": I18n.t(:text_user_wrote, value: ERB::Util.html_escape(journal.user)),
+                             test_selector: "op-wp-journal-#{journal.id}-quote"
+                           }
+                         }) do |item|
+            item.with_leading_visual_icon(icon: :quote)
+          end
+        end
+
+        def bubble_html
+          "
+          <span
+            class=\"comments-number--bubble op-bubble op-bubble_mini\"
+            data-test-selector=\"user-activity-bubble\"
+          ></span>
+          ".html_safe # rubocop:disable Rails/OutputSafety
+        end
```

Comment:
```suggestion
        def bubble_html
          content_tag(:span, "", class: "comments-number--bubble op-bubble op-bubble_mini", data: { "test-selector": "user-activity-bubble")
        end
```

Why not use a Primer::Beta::Counter for this? If you need to use an op-bubble, avoid using `.html_safe`.

Reply:
I'm using the `dot-fill` icon now